.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
We usually `.gitignore` this file to allow individual developers to use their own preferred settings. By default any settings in a project's `.rspec` will override those in a user's global `~/.rspec`.

This means that to make sure `spec_helper` is required everywhere, it needs to be explicitly required at the top of each spec.